packages/wct-mocha/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "wct-mocha",
+  "version": "1.0.0-pre.1",
+  "description": "Client-side library for testing web-components with Mocha.",
+  "main": "lib/index.js",
+  "files": [
+    "browser.js",
+    "lib/",
```

Comment:
Yes because wct-browser-legacy and web-component-tester are going to be dependent on wct-mocha's lib to build their own client-side distribution code.

Reply:
And if someone else wants to use wct-mocha to do same custom something-something they can.